apps/nowcasting-app/components/charts/gsp-pv-remix-chart/use-get-gsp-data.ts

```diff
@@ -25,7 +28,9 @@ const useGetGspData = (gspId: number) => {
 
```

Comment:
Is it more readable, if you declare. a variblae called `nMinuteForecast = nhourForecast * 60` and then use this in the url below?

Reply:
Can do 👌🏻